src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class=" calendar calendar--start-day-sun calendar--month-length-31 ">
```

Comment:
Delete some redundant spaces
```suggestion
    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```